router/schema.go

```diff
@@ -215,6 +215,15 @@ CREATE TRIGGER notify_route_certificates_update
 		`ALTER TABLE http_routes ADD COLUMN drain_backends boolean NOT NULL DEFAULT TRUE`,
 		`UPDATE http_routes SET drain_backends = false WHERE service = 'controller'`,
 	)
+	migrations.Add(7,
+		`ALTER TABLE http_routes ADD COLUMN port integer CHECK (port > -1 AND port < 65535)`,
+		`UPDATE http_routes SET port = 0 WHERE port IS NULL`,
```

Comment:
port should be `NOT NULL` with a default value of `0` (which avoids updating the old values)